server/athenian/api/__init__.py

```diff
@@ -32,6 +33,8 @@
 from athenian.api import metadata
 from athenian.api.auth import Auth0
 from athenian.api.connexion import AthenianApp
+from athenian.api.experiments.preloading.cache import build_memory_cache_options, \
```

Comment:
Let's move this code in the main tree, `experiments` is for other kinds of experiments :)